_summaries/wu2021rethinking_2.md

```diff
@@ -1,19 +1,71 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Rethinking "Batch" in BatchNorm, Wu, Johnnson; 2021
+paper: wu2021rethinking_2
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: saikm200022
+score: 7
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# **Summary - Rethinking "Batch" in BatchNorm, Wu, Johnnson; 2021**
+
+## What is the core idea?
+
+BatchNorm is a commonly used component of neural networks that has greatly increased performance in image classifcation tasks. However, there are certain inherent aspects of BatchNorm that can lead to sub-optimal performance if not addressed correctly. This paper identifies issues primarily around population statistics computation,  training and test set inconsistency, domain shift, and information leakage. The paper also suggests solutions to combat each of these issues to make the most out of BatchNorm in networks.
+
+![Alt Text](wu2021_1f.PNG)
+
+
+## How is it realized (technically)?
+
+There were 4 main experiments performed in this paper:
+
+1. Train a ResNet-50 model on 100 epochs with a mini-batch size = 8192. EMA (Exponential Moving Average) is used to calculate population statistics with lambda = 0.9. Randomly checked and compared the population mean and EMA mean of a random channel in a random BatchNorm Layer. They run a second experiment on a ResNet-50 model with PreciseBN as the population statistic computation method. 
+
+![Alt Text](wu2021_1b.PNG)
+
+2. Train a ResNet-50 model with varying normalization batch size from 2 to 1024. Error was inspected under 3 setttings - using mini-batch statistics on training set and validation set, and using population statistics on the validation set.
+
+3. Train a ResNet-50 model on ImageNet-C with a normalization batch size of 32. ImageNet-C has corrupted images which was used to induce domain shift. 
+
+4. Used the second stage of an R-CNN object detector that takes region-of-interets for each image and derives prediction for that ROI. BatchNorm is used in R-CNN where regions from all images are merged into one mini-batch. This model was implemented with a pre-trained ResNet-50. They tested information leakage with 2 scenarios: using SyncBN and shuffling ROI among GPUs prior to the second stage. 
+
+
+## How well does the paper perform?
+
+### **Population Statistic Computation - More Representative Population Statistics**
+
+The paper found that EMA was not successful in finding representative population or mini-batch statistics. As shown  with the discrepancy in the below graph:
+
+![Alt Text](wu2021_1a.PNG)
```

Comment:
Any potential reason mentioned in the paper why the discrepancy happens?

Reply:
It was, I have added the reasoning to the summary.